StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PollingViewController.swift

```diff
@@ -31,12 +31,11 @@ class PollingViewController: UIViewController {
     private let safariViewController: SFSafariViewController?
 
     private lazy var intentPoller: IntentStatusPoller = {
-        guard let currentAction = currentAction as? STPPaymentHandlerPaymentIntentActionParams,
-              let clientSecret = currentAction.paymentIntent?.clientSecret else { fatalError() }
+        guard let currentAction = currentAction as? STPPaymentHandlerPaymentIntentActionParams else { fatalError() }
```

Comment:
Should this be an unexpected error?

Reply:
I have another PR in waiting that will remove this fatalError